exporter/awsemfexporter/README.md

```diff
@@ -19,6 +19,11 @@ This exporter converts OpenTelemetry metrics to
 and then sends them directly to CloudWatch Logs using the 
 [PutLogEvents](https://docs.aws.amazon.com/AmazonCloudWatchLogs/latest/APIReference/API_PutLogEvents.html) API.
 
+---
+**_NOTE:_** The [batch](https://github.com/open-telemetry/opentelemetry-collector/tree/main/processor/batchprocessor) processor must be used with the EMF Exporter if there are more than 100 metrics per scrape as the log event only supports 100 metric definition objects.
```

Comment:
can we add the configuration for the batch processor that should be used? I suppose we need to use a batch processor and set the `send_batch_max_size` to 100. Is that correct?